### PR TITLE
[UserNotificationsUI] Update the fro Xcode 10.

### PR DESCRIPTION
**Pull request by mandel-macaque**
Diff can be found here: 

https://github.com/xamarin/xamarin-macios/wiki/UserNotificationsUI-iOS-xcode10-beta1

### CODE REVIEW

File: src/usernotificationsui.cs [hunk 1]
```diff
@@ -83,6 +83,23 @@ interface NSExtensionContext_UNNotificationContentExtension {
...
+
+		[NoWatch, NoTV, NoMac, iOS (12,0)]
+		[Export ("performNotificationDefaultAction")]
+		void PerformNotificationDefaultAction ();
+
+		[NoWatch, NoTV, NoMac, iOS (12,0)]
+		[Export ("dismissNotificationContentExtension")]
+		void DismissNotificationContentExtension ();
+
+		// property, but we have to add the two methods
```
Comment (rolfbjarne) [inline src/usernotificationsui.cs]: Maybe add "because this is a category" here.

Comment (mandel-macaque) [reply src/usernotificationsui.cs]: @rolfbjarne valid point, we never know who is going to read the code. Will push a small change with that.

File: src/usernotificationsui.cs [hunk 2]
```diff
@@ -83,6 +83,23 @@ interface NSExtensionContext_UNNotificationContentExtension {
...
+		[iOS (12, 0)]
+		[Export ("notificationActions", ArgumentSemantic.Copy)]
+		UNNotificationAction[] NotificationActions ();
+
+		[iOS (12, 0)]
+		[Export ("setNotificationActions:", ArgumentSemantic.Copy)]
```
Comment (dalexsoto) [inline src/usernotificationsui.cs]: Remove the ArgumentSemantic, not used.

**Comment by mandel-macaque** [comment]
Timeouts are not related to the PR.

**Pull request merged**